Improve RN entry (#3714)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-24.04/releases/centreon-commercial-extensions.mdx

  <summary>Bug fixes</summary>

- [Server] Fixed a synchronization issue that could cause resources to disappear from a map.

> - [Server] Optimized the computation of the global map status.

Author: ```suggestion
- [Server] Optimized the way the global map status is calculated.
```
